drivers/pulseaudio/audio_driver_pulseaudio.cpp

```diff
@@ -293,7 +293,17 @@ Error AudioDriverPulseAudio::init() {
 	pa_ml = pa_mainloop_new();
 	ERR_FAIL_COND_V(pa_ml == nullptr, ERR_CANT_OPEN);
 
-	pa_ctx = pa_context_new(pa_mainloop_get_api(pa_ml), "Godot");
+	String context_name;
+	if (Engine::get_singleton()->is_editor_hint()) {
+		context_name = "Godot Editor";
```

Comment:
`#include "core/version.h"` and use `VERSION_NAME` to avoid hardcoding the name "Godot".